service/storage_service.cc

```diff
@@ -731,8 +753,10 @@ void storage_service::join_token_ring(int delay) {
         maybe_start_sys_dist_ks();
         mark_existing_views_as_built();
         db::system_keyspace::update_tokens(_bootstrap_tokens).get();
+
         bootstrap();
         // bootstrap will block until finished
+
```

Comment:
It is trivial but they are unrelated changes. It is better not to be included in this patch.

Reply:
Fixd